File: 3429-SpecialArrayI/3429-SpecialArrayI.py
# Last updated: 31/07/2025, 21:05:06
class Solution:
    def isArraySpecial(self, nums: List[int]) -> bool:
        
        for i in range(len(nums)-1):
            if nums[i] % 2 == nums[i+1] % 2 :
                return False
        return True

File: 3429-SpecialArrayI/3429-SpecialArrayI.java
// Last updated: 31/07/2025, 21:05:01
class Solution {
    public boolean isArraySpecial(int[] nums) {
        for(int i=0;i<nums.length-1;i++)
        {
            if(nums[i]%2==nums[i+1]%2)
            {
                return false;
            }
        }
        return true;
    }
}